manifests/r/RubyInstallerTeam/RubyWithDevKit/3/2/3.2.0-1/RubyInstallerTeam.RubyWithDevKit.3.2.installer.yaml

```
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RubyInstallerTeam.RubyWithDevKit.3.2
+PackageVersion: 3.2.0-1
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent /allusers
+  SilentWithProgress: /silent /allusers
+UpgradeBehavior: install
+FileExtensions:
+- rb
+- rbw
+ReleaseDate: 2022-04-26
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/oneclick/rubyinstaller2/releases/download/rubyinstaller-3.2.0-1/rubyinstaller-devkit-3.2.0-1-x64.exe
+  InstallerSha256: D82313EED91F3210C3CFAC42214497477F6919A540C6CA0F237D25C3810040E2
+  AppsAndFeaturesEntries:
+  - DisplayName: Ruby 3.2.0-1-x64-ucrt with MSYS2
+    ProductCode: RubyInstaller-3.2-x64-mingw-ucrt_is1
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/oneclick/rubyinstaller2/releases/download/rubyinstaller-3.2.0-1/rubyinstaller-devkit-3.2.0-1-x86.exe
+  InstallerSha256: 891C0F05CDA258049C36ACEF1401B5DD19A7EFB277108D2170DB34C6770AE282
+  AppsAndFeaturesEntries:
+  - DisplayName: Ruby 3.2.0-1-x86-msvcrt with MSYS2
+    ProductCode: RubyInstaller-3.2-i386-mingw32_is1
```

```suggestion
    ProductCode: RubyInstaller-3.2-i386-mingw32_is1
- Architecture: x86
  InstallerType: inno
  InstallerUrl: https://github.com/oneclick/rubyinstaller2/releases/download/rubyinstaller-3.2.0-1/rubyinstaller-devkit-3.2.0-1-x86.exe
  InstallerSha256: 891C0F05CDA258049C36ACEF1401B5DD19A7EFB277108D2170DB34C6770AE282
  AppsAndFeaturesEntries:
  - DisplayName: Ruby 3.2.0-1-x86
    ProductCode: RubyInstaller-3.2-i386-mingw32_is1
  Scope: user
  InstallerSwitches:
    Custom: /CURRENTUSER
```
Another installer entry for user scope.